buildenv/jenkins/docker-nodes/jenkinsfile-build-container.groovy

```diff
@@ -62,7 +62,7 @@ if ("${BUILD_OPTS}" == "") {
 }
 
 def NODE = (params.NODE) ? params.NODE : "sw.tool.docker && hw.arch.${ARCH}"
-def NAMESPACE = (params.NAMESPACE) ? params.NAMESPACE : "eclipse-openj9"
+def NAMESPACE = (params.NAMESPACE) ? params.NAMESPACE : "openj9"
```

Comment:
This is the namespace at dockerhub, right?

Reply:
eclipseopenj9, not openj9? 
https://hub.docker.com/r/eclipseopenj9/jenkins-agent-x86-centos6

Reply:
Correct, I hadn't had the update from the bugzilla I opened to create that repo until earlier today.  I will correct this as well.

Reply:
Corrected to the location in the bugzilla with in my latest push.  I also fixed the REPOSITORY value for the same reason.